### PR TITLE
fixed https://github.com/NuGet/Home/issues/1087

### PR DESCRIPTION
**Pull request by zhili1208**
Uri.EscapeDataString will convert "+" to "%2B".
need to Covert back to "+" before return.

### CODE REVIEW

**Comment by zhili1208** [comment]
@yishaigalatzer @deepakaravindr

**Comment by yishaigalatzer** [comment]
Why not use the same replace pattern?

File: src/ProjectManagement/Utility/PathUtility.cs [hunk 1]
```diff
@@ -91,7 +91,8 @@ public static string GetPath(Uri uri)
         public static string GetPackagePath(string path)
```
Comment (deepakaravindr) [inline src/ProjectManagement/Utility/PathUtility.cs]: @zhili1208, Can you please git blame this file to understand the history of changes on this method? As I remember it, this method was changed more than once for fixing 2 or more different bugs. Please make sure that your fix does not cause any regressions

**Comment by zhili1208** [comment]
@emgarten

**Comment by deepakaravindr** [comment]
Can you please add tests for this? Thanks!

**Comment by emgarten** [comment]
This should work. PackageReader for zip files unescapes the paths from the start, so there is no need to guess the path like this was doing before. For PackageFolderReader which may come up during template installs this isn't done, but packages should be unescaped when extracted to disk from what I see.

I suggest looked through the packages in program files used by the templates to make sure there aren't any escaped paths, and if you find one check that it installs correctly.

**Comment by yishaigalatzer** [comment]
If you don't find one, please create a wizard project with a  (with + in the path) and test it this way

**Comment by borysrybak** [comment]
Hey :) sorry for interfering.
Here is my issue with exactly the same problem: http://stackoverflow.com/questions/31777604/cannot-install-mvvmcross-package-in-class-library-portable-pcl/

So you can use it as example package with '+' signs

**Comment by zhili1208** [comment]
@emgarten checked Template install, it still works in this case.

**Comment by zhili1208** [comment]
added function Test for this in https://github.com/NuGet/NuGet.VisualStudioExtension/commit/abeb7eae903ea5d27e1677211225b2dbe46dd070

**Pull request closed without merging**